src/content/docs/en/guides/cms/frontmatter-cms.mdx

```diff
@@ -0,0 +1,124 @@
+- Click on the first step to **Initialize project**;
+- As Astro is one of the supported frameworks, you can select it from the list;
+- Register your content folders, in this case the `src/content/blog` folder;
+- Click on **Show the dashboard** to open the content dashboard.
+
+### Project configuration
+
+Once the project is initialized, you will get a `frontmatter.json` configuration file and a `.frontmatter` folder in the root of your project.
+
+<FileTree title="Project Structure">
+- .frontmatter/
+  - database/
+    - mediaDb.json
+- src/
+- astro.config.mjs
+- **frontmatter.json**
+- package.json
+</FileTree>
+
+#### Content-type configuration
+
+Content-types is the way Front Matter CMS knows how to handle your content. You can configure the content-types in the `frontmatter.json` file. 
+
+- Open the `frontmatter.json` file;
+- Replace the `frontMatter.taxonomy.contentTypes` array with the following content-types configuration:
+
+```json title="frontmatter.json"
+{
+  "frontMatter.taxonomy.contentTypes": [
+    {
+      "name": "default",
+      "pageBundle": false,
+      "previewPath": "'blog'",
+      "filePrefix": null,
+      "fields": [
+        {
+          "title": "Title",
+          "name": "title",
+          "type": "string",
+          "single": true
+        },
+        {
+          "title": "Description",
+          "name": "description",
+          "type": "string"
+        },
+        {
+          "title": "Publishing date",
+          "name": "pubDate",
+          "type": "datetime",
+          "default": "{{now}}",
+          "isPublishDate": true
+        },
+        {
+          "title": "Content preview",
+          "name": "heroImage",
+          "type": "image",
+          "isPreviewImage": true
+        }
+      ]
+    }
+  ]
+}
+```
+
+:::note
+This configuration makes sure that the content-type matches the blog content its fields.
+:::
+
+### Preview your articles in the editor
+
+From the **Front Matter CMS** panel, click on the **Start server** button. This will start the Astro local dev server. Once running, you can open the content dashboard, select one of the articles and click on the **Open preview** button to open the article in the editor.
+
+![Front Matter CMS - Astro](/cms/frontmatter-cms/astro-preview-frontmatter-cms.png)
+
+### Create new articles
+
+Open the **Front Matter CMS Dashboard**, you can do this as follows: 
+
+- Using the keyboard binding: `alt+d` (Windows & Linux) or `option+d` (macOS);
+- Open the command palette and search for `Front Matter: Open dashboard`;
+- Click on the **Front Matter** icon on the title bar of the panel or files.
+
+![Open Front Matter CMS Dashboard](/cms/frontmatter-cms/frontmatter-open-dashboard.png)
+
+- Click on the **Create content** button;
+- Front Matter will ask you for the title of the article, fill it in and press enter;
+- Your new article will be created and opened in the editor. You can start writing your article.
+
+## Useful links
```

Comment:
Our other CMS pages include Official and Community Resources headings, I would rename this for consistency's sake, considering all the links here are the official site, docs, and a video by you, who maintains the project.
```suggestion
## Official Resources
```